ch_5_AppDev_Intermediate/5_1_ChatBot.ipynb

```diff
@@ -51,21 +49,18 @@
     "from huggingface_hub import snapshot_download\n",
     "\n",
     "model_path = snapshot_download(repo_id='meta-llama/Llama-2-7b-chat-hf',\n",
-    "                               token='hf_XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX') # change it to your own Hugging Face access token"
+    "                              token='hf_XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX') # change it to your own Hugging Face access token\n"
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "> **Note**\n",
     ">\n",
     "> The model will by default be downloaded to `HF_HOME='~/.cache/huggingface'`.\n",
     "\n",
-    "### 5.1.2.1 Load Model in Low Precision\n",
-    "\n",
-    "One common use case is to load a Hugging Face *transformers* model in low precision, i.e. conduct **implicit** quantization while loading.\n",
+    "  One common use case is to load a Hugging Face *transformers* model in low precision, i.e. conduct **implicit** quantization while loading.\n",
```

Comment:
should we remove `5.1.2.1 Load Model in Low Precision` section @shane-huang ? If remove, following section also need to modify.